### PR TITLE
Change presentationModeExit keybinding

### LINKED ISSUE #3: Ability to disable ESC for quitting presentation mode
Would be nice to have another option (or just by running the command again) to exit presentation mode.

the escape key may be pressed quite a lot, especially when dealing with intellisense

### PR DESCRIPTION
**Pull request by texastoland**
Fixes #3 by matching `exitZenMode` mode. Zen mode can still be exited using `toggleZenMode`.

### CODE REVIEW

**Comment by jspolancor** [comment]
> texastoland

I will take a look and make a few test tomorrow, I haven't touched the extension in years, maybe there's updates to make before merging this, Thanks for the contribution!

**Comment by texastoland** [comment]
No worries! This was just a config change but happy to pitch in the future. And thanks your extension should be built-in 🏆

**Review by keislamoglu — APPROVED** [review]

**Comment by keislamoglu** [comment]
I like coding in presentation mode since it allows me to focus more on what I do. However, I use very frequently the key "Esc" to close popups like parameter hints, etc. So, can't wait for this PR to be merged.

**Comment by jspolancor** [comment]
A quick update on this, I'm working on it right now, as I expected the dependencies and a few bits of the code need to be updated, I'll test and merge this PR after those fixes 🤗

**Comment by keislamoglu** [comment]
What is the last status? Can you give an update?

**Comment by jsmitty12** [comment]
I like this change, but there's no need to wait for a new version of the extension to make this change for yourself. Personally, I removed the keybinding, which forces me to open the command pallet and toggle Presentation Mode that way. Here's what I added to my `keybindings.json` file:

```json
{
    {
        "key": "escape",
        "command": "-extension.presentationModeExit",
        "when": "inPresentationMode"
    },
}
```

**Pull request closed without merging**
